cmass/conf/global.yaml

```diff
@@ -9,6 +9,7 @@ hydra:
         datefmt: "%H:%M:%S"
 
 meta:
-  wdir: "/home/mattho/git/ltu-cmass/data"
+  #wdir: "/home/mattho/git/ltu-cmass/data"
+  wdir: "/data101/bartlett/ili/cmass/"
```

Comment:
I may revert this later, before merging to main, to maintain consistency. However, in #36 there's a to-do to remove this file from change tracking.

Reply:
Done.